app/helpers/admin_helper.rb

```diff
@@ -1,5 +1,5 @@
 module AdminHelper
-  def thing_type(thing)
-    request.path.split('admin/').last.split("/").first.capitalize.singularize
+  def thing_type(_thing)
```

Comment:
Any reason you didn't just get rid of `_thing`? And maybe rename to `current_resource_name` or something.